ini/assault.ini

```diff
@@ -1,37 +1,41 @@
-[data]
-name     = Assault
-driver   = System2x
-type     = System2
-filename = at1snd0.bin
-gain     = 1.0
-muterear = 0
-
-[wave.0]
-filename = atvoi1.bin
-length   = 0x80000
-position = 0x100000
-
-;[wave.1]
-;filename = fl1-v2
-;length   = 0x20000
-;position = 0x300000
-
-[config]
-fm_volcalc = 1
-pcm_adsr = 1
-pcm_link = 1
-
-[playlist]
-813 = Opening
-12 = BGM 1
-13 = BGM 2
-22 = Lift Down
-17 = BGM 3
-18 = BGM 4
-21 = Lift Up
-26 = Ending
-23 = Name Entry 1
-20 = Name Entry 2
-0e = Unused 1
-10 = Unused 2
-820 = Speaker Check
+[data]
+name     = Assault
+driver   = System2x
+type     = System2
+filename = at1snd0.bin
+gain     = 1.0
+muterear = 0
+
+[wave.0]
+filename = atvoi1.bin
+length   = 0x80000
+position = 0x100000
+
+;[wave.1]
+;filename = fl1-v2
+;length   = 0x20000
+;position = 0x300000
+
+[config]
+fm_volcalc = 1
+pcm_adsr = 1
+pcm_link = 1
+
+[playlist]
+813 = Opening
+21 = Lift Down
+12 = BGM 1
+13 = BGM 2
+22 = Lift Up
+17 = BGM 3
+18 = BGM 4
+26 = Ending
+23 = Name Entry 1
+20 = Name Entry 2
+0e = Unused 1
+10 = Unused 2
+0c = Unused 3 C140

```

Comment:
It is possible to play both parts simultaneously by creating an action to start the subsongs and using a playlist trick to force the action to begin immediately. See lbgrande.ini